cirq-core/cirq/ops/raw_types.py

```diff
@@ -203,8 +203,10 @@ class Gate(metaclass=value.ABCMetaImplementAnyOneOf):
     to create an Operation on q1 and q2 is equivalent to MyGate(q1,q2).
 
     Gates operate on a certain number of qubits. All implementations of gate
-    must implement the `num_qubits` method declaring how many qubits they
-    act on.
+    must implement a method to declare the number of qubits (if a gate acting
```

Comment:
nit - ignore if it were ok before
```suggestion
    must implement a method to declare the number of qubits (if a gate is acting
```